libs/estdlib/src/erlang.erl

```diff
@@ -1493,3 +1494,12 @@ unique_integer() ->
 -spec unique_integer([monotonic | positive]) -> integer().
 unique_integer(_Options) ->
     erlang:nif_error(undefined).
+
+%%-----------------------------------------------------------------------------
+%% @param   Reason (usually undefined)
+%% @doc     Declare a NIF function, making Dialyzer happy.
+%% @end
+%%-----------------------------------------------------------------------------
+-spec nif_error(Reason :: any()) -> no_return().
```

Comment:
We may want to skip documentation for this function all together, it doesn’t seem to be included in OTP docs, at least I can’t find any reference to it.

Reply:
You are right.